defaults/main.yml

```
@@ -5,5 +5,20 @@ bareos_webui_configuration:
   - section: session
     option: timeout
     value: 3600
+  - section: "{{ bareos_webui_directors.name }}"
+    option: ca_file
+    value: "{{ bareos_webui_tls_ca_cert }}"
+  - section: "{{ bareos_webui_directors.name }}"
+    option: cert_file
+    value: "{{ bareos_webui_tls_cert_chain }}"
 
 bareos_webui_directors: []
+
+# Enable TLS
+bareos_webui_tls_enable: yes
```

use true/false for boolean values
```suggestion
bareos_webui_tls_enable: true
```

---

also the default value probably should be `false` as this would break otherwise, if you don't provide a CA and certificate.